internal/server/providers/tmux/tmux_e2e_test.go

```
@@ -0,0 +1,315 @@
+	socket = filepath.Join(dir, "s.sock")
+
+	// Spawn the server with a single dummy session so list-sessions
+	// returns at least one row immediately. `tmux new-session -d`
+	// daemonises the server; the -t target is unused.
+	args := []string{
+		"-S", socket,
+		"new-session", "-d", "-s", "alpha",
+		"-x", "80", "-y", "24",
+		"bash", "-c", "while true; do sleep 1; done",
+	}
+	cmd := exec.Command("tmux", args...)
```

golangci-lint (2.12.1)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)